src/components/ErrorPopup.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+// /* global window */
+
+export default (shortDescription, error) => {
+  const errorHandler = value => {
+    let body;
+    if (error instanceof Error) {
+      body = `
+      Description of the Error:
+      ${shortDescription}
+
+      Error name:
+      ${error.message}
+      
+      Error lineNumber:
+      ${error.lineNumber}
+      
+      Error fileName:
+      ${error.fileName}
+      
+      Error stack:
+      ${error.stack}
+    `;
+    } else {
+      body = `
+      Description of the Error:
+      ${shortDescription}
+
+      Transaction link:
+      ${error}
+    `;
+    }
+
+    if (value === 'email') {
+      window.open(`mailto:bugs@giveth.io?subject=Error in DApp&body=${encodeURIComponent(body)}`);
```

Comment:
Lets take the email out of here into the configuration with a default value bugs@giveth.io.